config/locales/pl.yml

```diff
@@ -228,6 +228,7 @@ pl:
     delete_account_html: Jeżeli chcesz usunąć konto, <a href="%{path}">przejdź tutaj</a>. Otrzymasz prośbę o potwierdzenie.
     didnt_get_confirmation: Nie otrzymałeś instrukcji weryfikacji?
     forgot_password: Nie pamiętasz hasła?
+    invalid_reset_password_token: Link resetowania hasła jest nieważny lub nieprawidłowy. Spróbuj ponownie.
```

Comment:
Maybe „Link do resetowania hasła”? Besides that, if link is either invalid or expired, then trying again seems to be pointless (this applies also to the English translation).